File: externe/src/SAVBundle/Entity/FullAdress.php
<?php

namespace SAVBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * FullAdress
 *
 * @ORM\Table(name="full_adress")
 * @ORM\Entity(repositoryClass="SAVBundle\Repository\FullAdressRepository")
 */
class FullAdress
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="rue", type="string", length=255)
     */
    private $rue;

    /**
     * @var int
     *
     * @ORM\Column(name="num_maison", type="integer")
     */
    private $numMaison;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=255)
     */
    private $ville;

    /**
     * @var int
     *
     * @ORM\Column(name="code_postal", type="integer")
     */
    private $codePostal;

    /**
     * @var string
     *
     * @ORM\Column(name="pays", type="string", length=255)
     */
    private $pays;


    /**
     *
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User")
     * @ORM\JoinColumn(name="idUser", referencedColumnName="id")
     */
    private $idUser;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set rue
     *
     * @param string $rue
     *
     * @return FullAdress
     */
    public function setRue($rue)
    {
        $this->rue = $rue;

        return $this;
    }

    /**
     * Get rue
     *
     * @return string
     */
    public function getRue()
    {
        return $this->rue;
    }

    /**
     * Set numMaison
     *
     * @param integer $numMaison
     *
     * @return FullAdress
     */
    public function setNumMaison($numMaison)
    {
        $this->numMaison = $numMaison;

        return $this;
    }

    /**
     * Get numMaison
     *
     * @return int
     */
    public function getNumMaison()
    {
        return $this->numMaison;
    }

    /**
     * Set ville
     *
     * @param string $ville
     *
     * @return FullAdress
     */
    public function setVille($ville)
    {
        $this->ville = $ville;

        return $this;
    }

    /**
     * Get ville
     *
     * @return string
     */
    public function getVille()
    {
        return $this->ville;
    }

    /**
     * Set codePostal
     *
     * @param integer $codePostal
     *
     * @return FullAdress
     */
    public function setCodePostal($codePostal)
    {
        $this->codePostal = $codePostal;

        return $this;
    }

    /**
     * Get codePostal
     *
     * @return int
     */
    public function getCodePostal()
    {
        return $this->codePostal;
    }

    /**
     * Set pays
     *
     * @param string $pays
     *
     * @return FullAdress
     */
    public function setPays($pays)
    {
        $this->pays = $pays;

        return $this;
    }

    /**
     * Get pays
     *
     * @return string
     */
    public function getPays()
    {
        return $this->pays;
    }

    /**
     * Set idUser
     *
     * @param \AppBundle\Entity\User $idUser
     *
     * @return FullAdress
     */
    public function setIdUser(\AppBundle\Entity\User $idUser = null)
    {
        $this->idUser = $idUser;

        return $this;
    }

    /**
     * Get idUser
     *
     * @return \AppBundle\Entity\User
     */
    public function getIdUser()
    {
        return $this->idUser;
    }
}

File: SoukLemdina_javaFX/src/SoukLemdina/Entities/FullAdress.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package SoukLemdina.Entities;

/**
 *
 * @author Lobna
 */
public class FullAdress {
    public int id ;
    public String rue;
    public int num_maison;
    public String ville;
    public int code_postal;
    public String pays;
    public int idUser;

    public FullAdress(String rue, int num_maison, String ville, int code_postal, String pays) {
        this.rue = rue;
        this.num_maison = num_maison;
        this.ville = ville;
        this.code_postal = code_postal;
        this.pays = pays;
    }

    public FullAdress(int i, String str0, int str1, String str2, int str3, String str4, int str5) {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getRue() {
        return rue;
    }

    public void setRue(String rue) {
        this.rue = rue;
    }

    public int getNum_maison() {
        return num_maison;
    }

    public void setNum_maison(int num_maison) {
        this.num_maison = num_maison;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public int getCode_postal() {
        return code_postal;
    }

    public void setCode_postal(int code_postal) {
        this.code_postal = code_postal;
    }

    public String getPays() {
        return pays;
    }

    public void setPays(String pays) {
        this.pays = pays;
    }

    public int getIdUser() {
        return idUser;
    }

    public void setIdUser(int idUser) {
        this.idUser = idUser;
    }

    @Override
    public String toString() {
        return "FullAdress{" + "rue=" + rue + ", num_maison=" + num_maison + ", ville=" + ville + ", code_postal=" + code_postal + ", pays=" + pays + '}';
    }
    
    
    
    
    
}
